src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import {createStore, applyMiddleware, combineReducers} from "redux";
```

Comment:
it's more common to name store folders more lib-agnostic ('src/store' for example) so in case we decide to switch from redux to mobx (or anything else), there will be less changes

Reply:
done